### PR TITLE
Add blog post for week 18

### PR DESCRIPTION
**Pull request by lbarcziova**

### CODE REVIEW

**Comment by lbarcziova** [comment]
Or should we skip this week?

**Review by majamassarini — APPROVED** [review]
I liked it 😄! But for me is the same.

**Comment by nforro** [comment]
> Or should we skip this week?

I'd skip it and do the next blogpost for week 18 and 19.

File: content/posts/weekly/May-2023.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Packit May 2023"
+date: 2023-05-10T08:41:12+02:00
+weight: 52
+---
+
+## Week 18 (May 4th - May 9th)
+
+During this short week (another Czech Republic public holiday) we didn't manage to implement
+any significant new features.
```
[the remark this answers is not in the document]
Comment (lbarcziova) [reply content/posts/weekly/May-2023.md:10]: > The issues with Bodhi. (And that people can retrigger.)

good point, will add this, thanks!

>   Last week (was it last week?), TF finally started supporting the merged state. I think it's worth mentioning.

so there is no further integration that needs to be added on our side?

[the remark this answers is not in the document]
Comment (lbarcziova) [reply content/posts/weekly/May-2023.md]: PTAL, added the note about Bodhi updates. As for TF, let's add it once verified it works (as for now doesn't work [here](https://github.com/packit/hello-world/pull/483#issuecomment-1541624560))

**Review by nforro — APPROVED** [review]

**Pull request merged**